README.md

```diff
@@ -13,3 +13,22 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+/////
+1
+(a) The amount of input data may be different, for example, n=1000 and n=1
```

Comment:
...and this is exactly what is captured in asymptotic analysis. It gives you an expression that shows how runtime changes with input size.